microshift_release_notes/microshift-4-17-release-notes.adoc

```diff
@@ -130,12 +131,12 @@ Red Hat Customer Portal user accounts must have systems registered and consuming
 
 This section is updated over time to provide notes on enhancements and bug fixes for future asynchronous errata releases of {microshift-short} {product-version}. Versioned asynchronous releases, for example with the form {microshift-short} {product-version}.z, are detailed in the following subsections.
 
-//TODO uncomment and verify info prior to merge freeze
-//[id="microshift-4-17-0-dp_{context}"]
-//=== RHEA-2024:3720 - {microshift-short} 4.17.0 bug fix and security update advisory
+//TODO verify info prior to merge
+[id="microshift-4-17-1-dp_{context}"]
+=== RHEA-2024:XXXX - {microshift-short} 4.17.1 bug fix and security update advisory
 
-//Issued: 2024-MM-DD
+Issued: DD Month 2024
 
-//{product-title} release 4.17.0 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHEA-2024:3720[RHEA-2024:3720] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHEA-2024:3718[RHEA-2024:3718] advisory.
+{product-title} release 4.17.1 is now available. Bug fixes and enhancements are listed in the link:https://access.redhat.com/errata/RHEA-2024:XXXX[RHEA-2024:XXXX] advisory. Release notes for bug fixes and enhancements are provided in this documentation. The images that are included in the update are provided by the {OCP} link:https://access.redhat.com/errata/RHEA-2024:XXXX[RHEA-2024:XXXX] advisory.
 
-//For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deploy_microshift-embed-in-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
\ No newline at end of file
+See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[Listing the contents of the {microshift-short} RPM release package].
```

Comment:
Maybe lowercase since the title's being used as part of the actual sentence.
```suggestion
See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
```

Reply:
TY! I wasn't sure about that cap because I know some are sticklers for putting exactly what is on the linked page there.